<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1525534279234" stop="1525534295292">
	<name>Test Login</name>
	<title>Test Login</title>
	<test-cases>
		<test-case start="1525534279235" stop="1525534295292" status="failed">
			<name>validate user credentials</name>
			<title>validate user credentials</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="c51f8f52-59e0-44e1-a73f-a420e2b24a25-attachment.png" type="image/png" size="47718"/>
			</attachments>
			<failure>
				<message>Failed: No element found using locator: By(xpath, //div[@id=&apos;passwordNext&apos;]//div[@class=&apos;ZFr60d CeoRYc&apos;])</message>
				<stack-trace>NoSuchElementError: No element found using locator: By(xpath, //div[@id=&apos;passwordNext&apos;]//div[@class=&apos;ZFr60d CeoRYc&apos;])
    at elementArrayFinder.getWebElements.then (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
    at ManagedPromise.invokeCallback_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at UserContext.&lt;anonymous&gt; (C:\Users\123\WebstormProjects\ProtractorTest\GmailExcelTest.js:11:85)
    at C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
From: Task: Run it(&quot;validate user credentials&quot;) in control flow
    at UserContext.&lt;anonymous&gt; (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
From asynchronous test: 
Error
    at Suite.&lt;anonymous&gt; (C:\Users\123\WebstormProjects\ProtractorTest\GmailExcelTest.js:8:5)
    at addSpecsToSuite (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\123\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.&lt;anonymous&gt; (C:\Users\123\WebstormProjects\ProtractorTest\GmailExcelTest.js:7:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>